proxy-node-acceptance-tests/.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.2
```

Comment:
Could we use 2.5.3?

Reply:
Yeah why not use the latest and greatest? We should bump the `pki` stuff while we're at it.

Reply:
The latest it is!